auth/app/src/main/java/com/google/firebase/quickstart/auth/PhoneAuthActivity.java

```diff
@@ -105,7 +105,8 @@ private void startPhoneNumberVerification(String phoneNumber) {
           PhoneAuthOptions.newBuilder(mAuth) 
               .setPhoneNumber(phoneNumber)       // Phone number to verify
               .setTimeout(60L, TimeUnit.SECONDS) // Timeout and unit
-              .setActivity(this)                 // Activity (for callback binding)
+              .setActivity(this)                 // Optional paramerter Activity (for callback binding). 
+                                                //  If no activity is passed, recaptcha verification can not be done.
```

Comment:
Maybe we could:
- remove the "parameter" word for simplicity
- make the newline comment have the same indentation as the builder
- say "used" instead of "done" to match what's said in [that page](https://firebase.google.com/docs/auth/android/phone-auth#enable-app-verification).

WDYT?

```suggestion
              .setActivity(this)                 // (optional) Activity for callback binding
              // If no activity is passed, reCAPTCHA verification can not be used.
```